storio-content-resolver/src/test/java/com/pushtorefresh/storio/contentresolver/operation/delete/DeleteObjectsStub.java

```diff
@@ -98,11 +97,23 @@ static DeleteObjectsStub newInstanceForDeleteMultipleObjectsWithTypeMapping() {
     }
 
     @NonNull
-    static DeleteObjectsStub newInstanceForDeleteOneObject() {
+    static DeleteObjectsStub newInstanceForDeleteOneObjectWithoutTypeMapping() {
         return new DeleteObjectsStub(false, 1);
     }
 
+    @NonNull
+    static DeleteObjectsStub newInstanceForDeleteOneObjectWithTypeMapping() {
+        return new DeleteObjectsStub(true, 1);
+    }
+
     void verifyBehaviorForDeleteMultipleObjects(@NonNull DeleteResults<TestItem> deleteResults) {
+        verify(storIOContentResolver).delete();
+
+        if (withTypeMapping || items.size() > 1) {
+            // should be called only once because of Performance!
+            verify(storIOContentResolver).internal();
+        }
```

Comment:
Yes! BTW, please don't merge before I'll fix such problems :) I don't want to do separate commits for that…